packages/runtime/src/interpreter/interpreter.ts

```diff
@@ -62,8 +62,8 @@ export class Interpreter {
 	// It is used to provide sub routine functionality
 	callStack: Stack<number>;
 	labelMap: Map<string, number>; // label string mapped to their respective indexes in instructions array
-	subTxn: EncTx | undefined; // "current" inner transaction
-	innerTxns: EncTx[]; // executed inner transactions
+	subTxn: EncTx[]; // "current" inner transaction
```

Comment:
1. shall we rename it to `currentInnerTx`?
2. Why this is a list? There can be only one current tx.

Reply:
> 1. shall we rename it to `currentInnerTxGroup`?

Oh, better name(y) Thanks @robert-zaremba!